### ISSUE TITLE
Expected CLASSDEFINITION (structure) @ RAP

### ISSUE THREAD

**Issue by lucasborin** [abaplint/abaplint#2584]
Could you please check this false-positive?

![image](https://user-images.githubusercontent.com/5233413/177333226-06774eb3-2263-4dc0-a3c0-fa9a4fb79ae0.png)

It is from the RAP framework.

**Comment by larshp** [comment]
can you add the code to reproduce the issue here? in text format please

**Comment by lucasborin** [comment]
```
CLASS cl_cpe_rmn_trm_conv_result_bp DEFINITION PUBLIC ABSTRACT FINAL FOR BEHAVIOR OF r_cmmdtyprcgconditionformulatp.
ENDCLASS.

CLASS cl_cpe_rmn_trm_conv_result_bp IMPLEMENTATION.
ENDCLASS.
```

**Comment by larshp** [comment]
what syntax version is configured in abaplint.json?

https://syntax.abaplint.org/?filter=classdef#/abap/statement/ClassDefinition

![image](https://user-images.githubusercontent.com/5888506/177337736-aaf0cdbc-c867-4023-946c-849e429ca72e.png)

**Comment by lucasborin** [comment]
Bingo: `"version": "v753"`. It is v755, isn't it?

**Comment by lucasborin** [comment]
Solved. Thank you Lars!
